prow/tide/history/history.go

```diff
@@ -45,19 +50,19 @@ type History struct {
 	sync.Mutex
 	logSizeLimit int
 
-	opener io.Opener
+	opener Opener
 	path   string
 }
 
-func readHistory(maxRecordsPerKey int, opener io.Opener, path string) (map[string]*recordLog, error) {
+func readHistory(maxRecordsPerKey int, opener Opener, path string) (map[string]*recordLog, error) {
 	reader, err := opener.Reader(context.Background(), path)
-	if io.IsNotExist(err) { // No history exists yet. This is not an error.
```

Comment:
Can we update `io` handle this scenario and continue using this package?

Reply:
Sorry there was another commit. I think this comment is outdated. After the latest commit there are no changes to history.go anymore